File: java-advent-of-code/src/test/java/advent/twenty_sixteen/Day24AirDuctSpelunkingTest.java
package advent.twenty_sixteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.*;

public class Day24AirDuctSpelunkingTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day24AirDuctSpelunking("/twenty_sixteen/Day24-AirDuctSpelunking-input.txt");

        assertEquals(470, dp.getPart1Answer().intValue());
        assertEquals(720, dp.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Integer, Integer> dp = new Day24AirDuctSpelunking("/twenty_sixteen/Day24-AirDuctSpelunking-example#1.txt");

        assertEquals(14, dp.getPart1Answer().intValue());
        assertEquals(20, dp.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_sixteen/Day24AirDuctSpelunkingTest.scala
package advent.twenty_sixteen

import org.scalatest.{FlatSpec, Matchers}

class Day24AirDuctSpelunkingTest  extends FlatSpec with Matchers {
  "2016 Day 24 - Input" should "calculate answers" in {
    val dp = new Day24AirDuctSpelunking("/twenty_sixteen/Day24-AirDuctSpelunking-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day24AirDuctSpelunking("/twenty_sixteen/Day24-AirDuctSpelunking-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
